addons/voice/org.openhab.voice.voicerss/src/main/java/org/openhab/voice/voicerss/internal/VoiceRSSTTSService.java

```diff
@@ -147,37 +146,37 @@ public AudioStream synthesize(String text, Voice voice, AudioFormat requestedFor
     }
 
     /**
-     * Initializes this.voices.
+     * Initializes voices.
      *
      * @return The voices of this instance
      */
-    private final HashSet<Voice> initVoices() {
-        HashSet<Voice> voices = new HashSet<Voice>();
+    private Set<Voice> initVoices() {
+        Set<Voice> voices = new HashSet<>();
         Set<Locale> locales = voiceRssImpl.getAvailableLocales();
```

Comment:
While you are editing this anyway we can change this too. There is no need to assign it to a variable but rather use it in the foreach loop directly:

for(Locale locale : voiceRssImpl.getAvailableLocales()) {
...
}